bin/hdfcoinc/pycbc_fit_sngls_binned

```diff
@@ -176,12 +179,19 @@ snr = snr[aboveminth]
 chisq = trigf[args.ifo+'/chisq'][:][aboveminth]
 chisq_dof = trigf[args.ifo+'/chisq_dof'][:][aboveminth]
 rchisq = chisq / (2 * chisq_dof - 2)
+
+try:
+    sgchisq = trigf[args.ifo+'/sg_chisq'][:][aboveminth]
+except:
```

Comment:
prefer to have 'except KeyError' or whatever will happen if you try this on a file with no sgchisq value.